specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.6/TextAnalytics.json

```diff
@@ -0,0 +1,3055 @@
+          "TWNationalID",
+          "TWPassportNumber",
+          "TWResidentCertificate",
+          "THPopulationIdentificationCode",
+          "TRNationalIdentificationNumber",
+          "UKDriversLicenseNumber",
+          "UKElectoralRollNumber",
+          "UKNationalHealthNumber",
+          "UKNationalInsuranceNumber",
+          "UKUniqueTaxpayerNumber",
+          "USUKPassportNumber",
+          "USBankAccountNumber",
+          "USDriversLicenseNumber",
+          "USIndividualTaxpayerIdentification",
+          "USSocialSecurityNumber",
+          "UAPassportNumberDomestic",
+          "UAPassportNumberInternational",
+          "Organization",
+          "Email",
+          "URL",
+          "Age",
+          "PhoneNumber",
+          "IPAddress",
+          "Date",
+          "Person",
+          "Address",
+          "All",
+          "Default"
+        ]
+      },
+      "type": "array",
+      "uniqueItems": true
+    },
+    "EntityResolutions": {
+      "properties": {
+        "dateTime.TimeRange": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TimeRangeResolution"
+          }
+        },
+        "dateTime.Time": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TimeResolution"
+          }
+        },
+        "quantity.Dimension": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DimensionResolution"
+          }
+        }
+      }
+    },
+    "TimeRangeResolution": {
+      "type": "object",
+      "properties": {
+        "timex": {
+          "type": "string",
+          "description": "timex expression representation"
+        },
+        "start": {
+          "type": "string",
+          "description": "beginning date/time of the interval"
+        },
+        "end": {
+          "type": "string",
+          "description": "ending date/time of the interval"
+        },
+        "confidenceScore": {
+          "type": "number",
+          "format": "double",
+          "description": "The probability of the resolution instance being correct for the extracted text."
+        }
+      }
+    },
+    "TimeResolution": {
+      "type": "object",
+      "properties": {
+        "timex": {
+          "type": "string",
+          "description": "timex expression representation"
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "time"
+          ],
+          "description": "The resolution type"
+        },
+        "value": {
+          "type": "string",
+          "description": "The actual time that the extracted text denote."
+        },
+        "confidenceScore": {
+          "type": "number",
+          "format": "double",
+          "description": "probability of the resolution instance being correct for the extracted text."
+        }
+      }
+    },
+    "DimensionResolution": {
+      "type": "object",
+      "properties": {
+        "subtype": {
```

Comment:
Should `subtype` and `unit` also be enums?

Reply:
Good point. Yes, I should be able to find the exhaustive list.